Dockerfile

```diff
@@ -32,12 +41,7 @@ RUN
     && find "$INSTALL_DIR/include" -type f -exec chmod +r {} \;                    \
     && rm "$PB_FILE"
 
-# The docker image is built as root - make binaries available to user.
-RUN mv /root/go/bin/* /usr/local/bin/
-
-ENV GOPATH=/agent/go
-ENV GOCACHE=/agent/.cache
-
-RUN echo "export PATH=\"\$PATH:\$(go env GOPATH)/bin\"" >> ~/.bashrc
+# Append to /etc/profile for login shells
+RUN echo 'export PATH="/usr/local/go/bin:$PATH"' >> /etc/profile
```

Comment:
In the debian docker in via the file `/etc/profile` the environment variable is overwritten with this approach:
```
if [ "$(id -u)" -eq 0 ]; then
  PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
else
  PATH="/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games"
fi
export PATH
```

Appending `export PATH ...` here makes sure, that the Go executable is available for every user of this docker image.